addon/visit.js

```diff
@@ -148,15 +171,17 @@ class PageVisitor {
         const timer = event.detail.timer;
         const tab = (await browser.tabs.get(event.detail.tabId));
 
-        this.setCurrentStatus({message: 'freezing'});
+        this.setCurrentStatus({message: 'freezing', index: tab.id});
```

Comment:
This status confuses me for the Vectorizer. We aren't freezing the page at any point here. I understand we're inheriting this from another class. Can we default to something else in this situation? Maybe "loading"?

Reply:
That is a good point. "freezing" makes sense for the corpus collector, but not the vectorizer. I could change it to something generic like "processing" ("loading" is already used for loading the actual page), or move this line into `processWithinTimeout()` and keeping "freezing" for the corpus collector and switch to "vectorizing" for the vectorizer. I'm leaning toward the latter. What do you think?

Reply:
Sure that sounds great.